plugins/modules/data_input_monitor.py

```diff
@@ -41,7 +41,6 @@
       - If set to C(True), the index value is checked to ensure that it is the name of a valid index.
     required: False
     type: bool
-    default: False
```

Comment:
why have we updated changes for the older module, was it due to fix some sanity failures, and if Not plz revert all of the older module changes